sdks/java/io/cdc/src/test/java/org/apache/beam/io/cdc/DebeziumIOConnectorTest.java

```diff
@@ -69,6 +103,7 @@ public void testDebeziumIOMySql() {
       );
 
 	  p.run().waitUntilFinish();
```

Comment:
Use `PAssert` to verify some records are consumed according to the data setup in MySql DB